### ISSUE TITLE
issue with sentiments dataset in get_sentiments

### ISSUE THREAD

**Issue by carlganz** [juliasilge/tidytext#64]
Hello,

I find that the `get_sentiments` function only works if `tidytext` is loaded, because of the `sentiments` dataset.

MRE:

```{R}
# tidytext not loaded
tidytext::get_sentiments("bing")
#errors
library(tidytext)
tidytext::get_sentiments("bing")
# works fine
```

I believe if https://github.com/juliasilge/tidytext/blob/master/R/sentiments.R#L71 is changed to use `tidytext::sentiments` it will work.

Kind Regards,
Carl

**Comment by juliasilge** [comment]
I believe that we have fixed this in #50, and thus this works correctly in the current development version but not yet on CRAN. Can you `devtools::install_github("juliasilge/tidytext")` and check it out?

(We are planning a CRAN release within the next month.)

**Comment by carlganz** [comment]
Yep it works on github. Thanks
